internal/csi-addons/rbd/volumegroup.go

```diff
@@ -115,7 +115,7 @@ func (vs *VolumeGroupServer) CreateVolumeGroup(
 			err.Error())
 	}
 
-	log.DebugLog(ctx, fmt.Sprintf("VolumeGroup %q had been created", req.GetName()))
+	log.DebugLog(ctx, fmt.Sprintf("VolumeGroup %q had been created: %+v", req.GetName(), vg))
```

Comment:
```
	log.DebugLog(ctx, fmt.Sprintf("VolumeGroup %q has been created: %+v", req.GetName(), vg))
Comment
```

Reply:
@Rakshith-R, sorry, I don't understand what you mean by this?

The `fmt.Sprintf()` call can be dropped, is that it?